src/modules/ie-objects/helpers/limit-access-to-object-details.spec.ts

```diff
@@ -30,6 +35,10 @@ describe('Limit access to object details', () => {
 				isKeyUser: true,
 				sector: IeObjectSector.PUBLIC,
 				accessibleVisitorSpaceIds: [],
+				licensesByUserGroup:
+					IE_OBJECT_LICENSES_BY_USER_GROUP[
+						Group.VISITOR ?? IeObjectExtraUserGroupType.ANONYMOUS
```

Comment:
now you're checking if an enum value exists and if not defaulting to anonymous
shouldn't you check the user's group instead of a static enum?